File: Easy/JavaScript/2806. Account Balance After Rounded Purchase.js
var accountBalanceAfterPurchase = function(purchaseAmount) {
    let n = purchaseAmount % 10;
    let bal = 0;

    if(n < 5)
        bal = purchaseAmount - n;
    else
        bal = purchaseAmount + 10 - n;
    
    return 100 - bal;
};

File: Easy/Python/2806. Account Balance After Rounded Purchase.py
class Solution(object):
    def accountBalanceAfterPurchase(self, purchaseAmount):
        n = purchaseAmount % 10
        bal = 0

        if n < 5:
            bal = purchaseAmount - n
        else:
            bal = purchaseAmount + 10 - n
        
        return 100 - bal
